src/main/java/com/iota/iri/service/validation/impl/TransactionSolidifierImpl.java

```diff
@@ -216,6 +280,12 @@ private void addToBroadcastQueue(TransactionViewModel tvm) {
         }
     }
 
+    @VisibleForTesting
+    Set<Hash> getSolidificationQueue(){
```

Comment:
It is funny that you call it `getQueue` but you return a `Set`

Reply:
A fair point, i'll rename